app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt

```diff
@@ -62,7 +67,15 @@ class HomeFragment : Fragment(R.layout.home_fragment) {
                     }
                 }
             addItemDecoration(HomeItemDecoration())
-            addOnScrollListener(ScrollPositionListener(loggedInViewModel::onScroll))
+            addOnScrollListener(
+                ScrollPositionListener(
+                    loggedInViewModel::onScroll,
+                    viewLifecycleOwner
+                )
```

Comment:
How about we do this?

```suggestion
                ScrollPositionListener(
                    { scrollPosition ->
                        scroll = scrollPosition
                        loggedInViewModel.onScroll(scrollPosition)
                    }},
                    viewLifecycleOwner
                )
```

That way we don't have to add two scroll listeners 🙂